### ISSUE TITLE
`TypeError` exception when running `seed` script

### ISSUE THREAD

**Issue by fPolic** [medusajs/medusa#1123]
Running the `seed` script will cause the following error: 
```
TypeError: Cannot read properties of undefined (reading 'constructor')
    at /Users/franepolic/Projects/medusa/store/node_modules/typeorm/entity-manager/EntityManager.js:726:118
    at Array.find (<anonymous>)
    at EntityManager.getCustomRepository (/Users/franepolic/Projects/medusa/store/node_modules/typeorm/entity-manager/EntityManager.js:725:103)
    at RegionService.<anonymous> (/Users/franepolic/Projects/medusa/store/node_modules/@medusajs/medusa/dist/services/region.js:323:54)
    at step (/Users/franepolic/Projects/medusa/store/node_modules/@medusajs/medusa/dist/services/region.js:48:23)
    at Object.next (/Users/franepolic/Projects/medusa/store/node_modules/@medusajs/medusa/dist/services/region.js:29:53)
    at /Users/franepolic/Projects/medusa/store/node_modules/@medusajs/medusa/dist/services/region.js:23:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/franepolic/Projects/medusa/store/node_modules/@medusajs/medusa/dist/services/region.js:19:12)
    at RegionService.validateFields_ (/Users/franepolic/Projects/medusa/store/node_modules/@medusajs/medusa/dist/services/region.js:315:16)
    at RegionService.<anonymous> (/Users/franepolic/Projects/medusa/store/node_modules/@medusajs/medusa/dist/services/region.js:175:63)
    at step (/Users/franepolic/Projects/medusa/store/node_modules/@medusajs/medusa/dist/services/region.js:48:23)
    at Object.next (/Users/franepolic/Projects/medusa/store/node_modules/@medusajs/medusa/dist/services/region.js:29:53)
    at /Users/franepolic/Projects/medusa/store/node_modules/@medusajs/medusa/dist/services/region.js:23:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/franepolic/Projects/medusa/store/node_modules/@medusajs/medusa/dist/services/region.js:19:12)
```
    
    
Also, this will cause that during project initialization with `medusa new my-medusa-store --seed` command, the main process is stuck on the seed step.

**Comment by adrien2p** [comment]
If you want to use the seed method while creating you must have an env file or specify the variable in the command, you can have a look at https://github.com/medusajs/medusa/blob/806c42650cc48eaec790a9009705d5d51683e914/packages/medusa-cli/src/create-cli.js#L146

Otherwise can you do
```bash
medusa new my-medusa-store
cd my-medusa-store
medusa seed  -m -f your-file.json
```

**Comment by fPolic** [comment]
Creating the project with the `--seed` flag should execute the equivalent command you suggested.

If I run the command directly, the same exception is thrown:
<img width="1088" alt="Screenshot 2022-02-27 at 15 22 11" src="https://user-images.githubusercontent.com/16856471/155894069-0060e3f8-3370-4b82-8290-d4768804da88.png">

**Comment by vavdav** [comment]
The same issue for me, when I am following this guide: https://docs.medusajs.com/tutorial/creating-your-medusa-server#setup-a-medusa-project, it stuck on 'Seeding database'

**Comment by olivermrbl** [comment]
Solved #1126
